### PR TITLE
Remove references to when old features became available

### PR DESCRIPTION
**Pull request by btat**
## Description

We often highlight when a new feature became available in a patch version X for minor version A, but this information is no longer relevant for subsequent minor versions (e.g. A+1, A+2,...). This PR removes such references.

### CODE REVIEW

**Review by sunilarjun — APPROVED** [review]
LGTM

File: docs/integrations-in-rancher/monitoring-and-alerting/windows-support.md [hunk 1]
```diff
@@ -6,8 +6,6 @@ title: Windows Cluster Support for Monitoring V2
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/monitoring-and-alerting/windows-support"/>
 </head>
 
-_Available as of v2.5.8_
-
 Starting at Monitoring V2 14.5.100 (used by default in Rancher 2.5.8), Monitoring V2 can now be deployed on a Windows cluster and will scrape metrics from Windows nodes using [prometheus-community/windows_exporter](https://github.com/prometheus-community/windows_exporter) (previously named `wmi_exporter`).
```
Comment (martyav) [inline docs/integrations-in-rancher/monitoring-and-alerting/windows-support.md:9]: Should we also update this line?

File: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters.md [hunk 2]
```diff
@@ -184,8 +184,6 @@ To prevent issues when upgrading, the [Kubernetes upgrade best practices](https:
 
 ## Authorized Cluster Endpoint Support for RKE2 and K3s Clusters
 
-_Available as of v2.6.3_
-
 Authorized Cluster Endpoint (ACE) support has been added for registered RKE2 and K3s clusters. This support includes manual steps you will perform on the downstream cluster to enable the ACE. For additional information on the authorized cluster endpoint, click [here](../manage-clusters/access-clusters/authorized-cluster-endpoint.md).
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters.md:187]:
```suggestion
Rancher supports Authorized Cluster Endpoints (ACE) for registered RKE2 and K3s clusters. This support includes manual steps you will perform on the downstream cluster to enable ACE. For additional information on the authorized cluster endpoint, click [here](../manage-clusters/access-clusters/authorized-cluster-endpoint.md).
```

File: versioned_docs/version-2.7/integrations-in-rancher/monitoring-and-alerting/windows-support.md [hunk 3]
```diff
@@ -6,8 +6,6 @@ title: Windows Cluster Support for Monitoring V2
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/monitoring-and-alerting/windows-support"/>
 </head>
 
-_Available as of v2.5.8_
-
 Starting at Monitoring V2 14.5.100 (used by default in Rancher 2.5.8), Monitoring V2 can now be deployed on a Windows cluster and will scrape metrics from Windows nodes using [prometheus-community/windows_exporter](https://github.com/prometheus-community/windows_exporter) (previously named `wmi_exporter`).
```
Comment (martyav) [inline versioned_docs/version-2.7/integrations-in-rancher/monitoring-and-alerting/windows-support.md:9]: Similar to an earlier comment, should we update this line to remove mention of when V2 was added?

**Review by martyav — CHANGES_REQUESTED** [review]
There were a few instances where we still mention versioning info under the removed lines. Should we also remove or keep?

File: docs/integrations-in-rancher/monitoring-and-alerting/windows-support.md (hunk 1 above)
Comment (btat) [reply docs/integrations-in-rancher/monitoring-and-alerting/windows-support.md]: Updated in c2585bf9c2b445a67a1649c6cae4d1cf41b890c7.

File: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters.md (hunk 2 above)
Comment (btat) [reply docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters.md]: Suggestion applied to applicable versions in c2585bf9c2b445a67a1649c6cae4d1cf41b890c7.

File: versioned_docs/version-2.7/integrations-in-rancher/monitoring-and-alerting/windows-support.md (hunk 3 above)
Comment (btat) [reply versioned_docs/version-2.7/integrations-in-rancher/monitoring-and-alerting/windows-support.md]: Updated in c2585bf9c2b445a67a1649c6cae4d1cf41b890c7.

**Review by sunilarjun — APPROVED** [review]
Thanks Billy!

**Review by martyav — APPROVED** [review]

**Pull request merged**